api/spec/src/app/app.tsp

```diff
@@ -0,0 +1,113 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/openapi3";
+
+import "./app.tsp";
+import "./capability.tsp";
+import "../errors.tsp";
+import "../oauth.tsp";
+import "../pagination.tsp";
+import "./stripe.tsp";
+import "../types.tsp";
+
+using OpenMeter.App;
+
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+
+namespace OpenMeter.App;
+
+/**
+ * Installed Apps API.
+ */
+@route("/api/v1/apps")
+@tag("Apps")
+interface Installed {
+  /**
+   * List apps.
+   */
+  @get
+  @route("/")
+  @operationId("listApps")
+  listApps(...ListAppsRequest): Paginated<App>[] | CommonErrors;
+
+  /**
+   * Get the app.
+   */
+  @get
+  @route("/{id}")
+  @operationId("getApp")
+  getApp(@path id: ULID): App | CommonErrors;
+
+  /**
+   * Uninstall an app.
+   */
+  @delete
+  @route("/{id}")
+  @operationId("uninstallApp")
+  uninstallApp(@path id: ULID): App | NotFoundError | CommonErrors;
+}
+
+/**
+ * Query params for listing installed apps
+ */
+@friendlyName("ListAppsRequest")
+model ListAppsRequest extends PaginatedQuery {}
```

Comment:
Should probably use `is`?

Reply:
Using`is` is helpful to mark the type the exact same. This is true in its current state, but I'm using extend because I am planning to add filters like is deleted, app type, etc.